LanguageFeatures/Extension-types/exhaustiveness_enum_A02_t01.dart

```diff
@@ -22,9 +22,10 @@ extension type ET1<T>(E<T> _) {}
 extension type ET2<T>(E<T> _) implements E<T> {}
 
 String testStatement1<T extends num>(ET1<T> e) {
+//     ^^^^^^^^^^^^^^
+// [analyzer] unspecified
   switch (e) {
 //^^^^^^
-// [analyzer] unspecified
 // [cfe] unspecified
```

Comment:
Same for a bunch of other cases.

Reply:
NB: This means that we're _accepting the discrepancy for now_ even though discrepancies are always a TODO. Should be OK because we have an issue: https://github.com/dart-lang/co19/issues/2439.